.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     # rocksdb, which are needed to build and test those modules.
     # To update the container image, see docker.yml.
     runs-on: ubuntu-latest
-    container: tendermintdev/docker-tm-db-testing
+    container: ghcr.io/faddat/cosmos
```

Comment:
reasoning?

Reply:
reasoning:  when we bump rocksdb, we also need to use a different docker image.  The alternative is to upgrade the current docker image.